src/app/index.html

```diff
@@ -7,6 +7,9 @@ <h2>
   <li>
     <a skyAppLink="/docs/radio">Radio</a>
   </li>
+  <li>
+    <a skyAppLink="/docs/toggle-switch">Toggle Switch</a>
```

Comment:
Probably doesn't matter much, but `switch` should be lowercase to follow our sentence-style capitalization pattern.

Reply:
✔️